component-catalog-connectors/artifactory-connector/setup.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2018-2022 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+from setuptools import find_packages, setup
+
+long_desc = """
+            Elyra component catalog connector for Artifactory
+            """
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+version_ns = {}
+with open(os.path.join(here, "artifactory_catalog_connector", "_version.py")) as f:
+    exec(f.read(), {}, version_ns)
+
+setup_args = dict(
+    name="elyra-artifactory-catalog-connector",
+    version=version_ns["__version__"],
+    url="https://github.com/elyra-ai/examples/tree/master/component-catalog-connectors/artifactory-connector",
```

Comment:
```suggestion
    url="https://github.com/elyra-ai/examples/tree/main/component-catalog-connectors/artifactory-connector",
```

Reply:
Resolved in https://github.com/elyra-ai/examples/pull/99/commits/b97b758ec6867f83c646bb560d12cfb80f7ee7bb